session/txn.go

```diff
@@ -77,6 +80,33 @@ func (st *TxnState) String() string {
 	return "invalid transaction"
 }
 
+// GoString implements the "%#v" format for fmt.Printf.
+func (st *TxnState) GoString() string {
+	var s strings.Builder
+	s.WriteString("Txn{")
+	if st.pending() {
+		s.WriteString("state=pending")
+	} else if st.Valid() {
+		s.WriteString("state=valid")
+		fmt.Fprintf(&s, ", startTS=%d", st.Transaction.StartTS())
+		if len(st.dirtyTableOP) > 0 {
+			fmt.Fprintf(&s, ", len(dirtyTable)=%d", len(st.dirtyTableOP))
+		}
+		if len(st.mutations) > 0 {
+			fmt.Fprintf(&s, ", len(mutations)=%d", len(st.mutations))
```

Comment:
I think print `st.mutations`'s content is useful, we can see `kind, tid, handle int64` but this PR miss them???

Reply:
So this is a dilemma:
when logs too much, you will think they are totally a mess
when logs too little, you find it hard to debug

Reply:
print len(mutations) is useless?

Reply:
`len(st.mutations) > 0` we know how many things are changed.
If we print mutation data, it maybe too much. @winkyao